Formula/x/x-cmd.rb

```diff
@@ -0,0 +1,26 @@
+class XCmd < Formula
+  desc "Bootstrap 1000+ command-line tools in seconds"
+  homepage "https://x-cmd.com"
+  url "https://raw.githubusercontent.com/x-cmd/release/main/dist/v0.5.5/allinone.tgz"
```

Comment:
Can we use https://github.com/x-cmd/x-cmd/archive/refs/tags/v0.5.5.tar.gz instead?